.github/workflows/run-solverdummy.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Install Dependencies
         run: |
           apt-get -qq update
-          apt-get -qq install software-properties-common python3-dev python3-pip git apt-utils
+          apt-get -qq install software-properties-common python3-dev python3-pip git apt-utils pkg-config
```

Comment:
Why don't you use consequently `precice/precice:2.4.0` here as well? I'm a `precice/precice` fan, but I'm still wondering why you are making the difference here.